cypress/e2e/spec.cy.js

```diff
@@ -76,32 +76,69 @@ describe('Register Tests', () => {
     cy.get('[placeholder="Email"]').type('email@email.com')
     cy.get('[placeholder="Password"]').type('password')
 
-    cy.intercept(`*register`, {
+    cy.intercept(`*user`, {
       statusCode: 403,
     })
 
-    cy.contains("Let's go!").click()
-
-    cy.on('windows:alert', (str) => {
-      expect(str).to.equal('Username or Email already taken, or invalid inputs')
-    })
+    const stub = cy.stub()
+    cy.on('window:alert', stub)
+    cy.contains("Let's go!")
+      .click()
+      .then(() => {
+        expect(stub.getCall(0)).to.be.calledWith(
+          'Username or Email already taken, or invalid inputs'
+        )
+      })
   })
 })
 
 describe('Login Tests', () => {
-  it('should login', () => {
+  /*it('should login', () => {
     cy.visit(Cypress.env('BASE_URL'))
 
+    cy.contains('UnReveal').should('be.visible')
+    cy.contains('Login').should('be.visible')
+    cy.contains('Register').should('be.visible')
+
     cy.contains('Login').click()
 
     cy.get('[placeholder="Email"]').type('email@email.com')
     cy.get('[placeholder="Password"]').type('password')
 
     cy.contains("Let's go!").click()
 
-    cy.on('windows:alert', (str) => {
-      expect(str).to.equal('Account created!')
+    cy.intercept(`*user`, {
+      statusCode: 201,
+    })
+
+    const stub = cy.stub()
+    cy.on('window:alert', stub)
+    cy.contains("Let's go!")
+      .click()
+      .then(() => {
+        expect(stub.getCall(0)).to.be.calledWith(
+          'Account created!'
+        )
+      })
+  })*/
+
+  it('should login', () => {
+    cy.visit(Cypress.env('BASE_URL'))
+
+    cy.contains('UnReveal').should('be.visible')
+    cy.contains('Login').should('be.visible')
+    cy.contains('Register').should('be.visible')
+
+    cy.intercept(`*login`, {
+      statusCode: 201,
     })
+
+    cy.contains('Login').click()
+
+    cy.get('[placeholder="Email"]').type('email@email.com')
+    cy.get('[placeholder="Password"]').type('password')
+
+    cy.contains("Let's go!").click()
```

Comment:
Should test if it is on main screen.
```suggestion
    cy.contains("Let's go!").click()
    cy.contains('UnReveal').should('be.visible')
    cy.contains('Login').should('be.visible')
    cy.contains('Register').should('be.visible')
```